process/etl/file_extract.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from .file_transform import *
+import os
+
+def extract_wrapper(
+        filepaths: List[str], apache_tika_jar: str,
```

Comment:
I have an idea. We can accept an Iterable instead of a list. Thus, the user can pass lists, tuples, dictionaries, etc. 

```
>>> def test(i):
...     for x in i:
...         print(x)
... 
>>> test([1,2,3])
1
2
3
>>> test((1,2,3))
1
2
3
>>> test({"a":1,"b":2,"c":3})
a
b
c
```

Reply:
great suggestion, i'll change the method to reflect this as well. tks